juliomontes-juliocmontes/hw_3/car_lot.rb

```diff
@@ -0,0 +1,69 @@
+require 'pry'
+require 'pry-byebug'
+
+
+def car_data
+  car_data = []
+  mustang = { name: "ford", model: "mustang", year: 2017, price: 30000, color: "black", is_electric: false }
+  f150 = { name: "ford", model: "f150", year: 2017, price: 40000, color: "blue", is_electric: false }
+  charger = { name: "dodge", model: "charger", year: 2014, price: 28000, color: "red", is_electric: false }
+  challenger = {name: "dodge", model: "challenger", year: 2017, price: 35000, color: "red", is_electric: false }
+  model_x = { name: "tesla", model: "model x", year: 2014, price: 50000, color: "black", is_electric: true }
+  corolla = { name: "toyota", model: "corolla", year: 2010, price: 16000, color: "red", is_electric: false }
+  camry = { name: "toyota", model: "camry", year: 2002, price: 3500, color: "blue", is_electric: false }
+
+  car_data.push(mustang,f150,charger,challenger,model_x,corolla,camry)
+  car_data
+end
+
+def user_requirements
+  puts "We have the following colors, please choose one: Blue, Black, Red, "
+  get_color = gets.strip.downcase
+  # puts "Do you prefer the car to be electric?"
+  # electrical = gets.strip
+  puts "What's your budget look like? (Please enter a number with no comma)"
+  price_range = gets.strip.to_i
+  puts "We have these brands, "
+  print_brand
+end
+
+def user_input
+  puts "Welcome to Ruby's Car Lot, would you like to check our inventory? "
+  check_inventory = gets.strip.downcase
+  case check_inventory
+  when "yes", "yeah", "sure"
```

Comment:
Haha, thanks. I only wish I could make it work. I'm reading up and trying to catch up. 

Sent from my iPhone

> On May 11, 2016, at 3:41 PM, Nicole Borgaard <notifications@github.com> wrote:
> 
> In juliomontes-juliocmontes/hw_3/car_lot.rb:
> 
> > +def user_requirements
> > +  puts "We have the following colors, please choose one: Blue, Black, Red, "
> > +  get_color = gets.strip.downcase
> > +  # puts "Do you prefer the car to be electric?"
> > +  # electrical = gets.strip
> > +  puts "What's your budget look like? (Please enter a number with no comma)"
> > +  price_range = gets.strip.to_i
> > +  puts "We have these brands, "
> > +  print_brand
> > +end
> > +
> > +def user_input
> > +  puts "Welcome to Ruby's Car Lot, would you like to check our inventory? "
> > +  check_inventory = gets.strip.downcase
> > +  case check_inventory
> > +  when "yes", "yeah", "sure"
> It's minor, but this little detail in allowing slightly different responses always makes me happy when using other people's programs. 😜
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>